services/doctor.services.js

```diff
@@ -0,0 +1,57 @@
+const { MedicaError } = require("../exceptions");
+const db = require("../models");
+
+exports.createDoctor = async ({
+  image,
+  title,
+  firstName,
+  lastName,
+  practiceArea,
+  adress,
+  city,
+  zip,
+  country,
+  phoneNumber,
+  website,
+  workingHours,
+  email,
+}) => {
+  if ((await db.Doctor.findOne({ where: { email } })) != null) {
```

Comment:
https://codeahoy.com/javascript/2019/10/12/==-vs-===-in-javascript/

Use `===`

Reply:
ok